package.json

```diff
@@ -36,6 +36,7 @@
     "eslint": "^2.10.2",
     "istanbul": "^0.4.3",
     "mocha": "^2.5.3",
+    "proxyquire": "^1.7.9",
```

Comment:
I know we have but I'm not really happy with the previous merged PR and this solution.  By doing it this way, the test has a hard dependency on the hmpo-form-controller module, also it isn't immediately clear what is happening, it only works as the result of the first require call is cached, which seems too automagical for unit tests IMO. @JoeChapman @easternbloc what are your thoughts on this vs proxyquire?